lua/lint/linters/clippy.lua

```diff
@@ -0,0 +1,51 @@
+local severities = {
+  note = vim.diagnostic.severity.INFO,
+  warning = vim.diagnostic.severity.WARN,
+  help = vim.diagnostic.severity.HINT,
+}
+
+local function parse(diagnostics, file_name, item)
+  for _, span in ipairs(item.spans) do
+    if span.file_name == file_name then
+      local message = item.message
+      if span.suggested_replacement ~= vim.NIL then
+        message = message .. " (" .. tostring(span.suggested_replacement) .. ")"
+      end
```

Comment:
```suggestion
      if span.suggested_replacement ~= vim.NIL then
        message = message .. "\nSuggested replacement:\n\n" .. tostring(span.suggested_replacement)
      end
```

Not sure if that's better. I can also merge it as is.